### PR TITLE
Tidying up the Interacting section

### PR DESCRIPTION
**Pull request by suearkinunity**
### Purpose of this PR

Cleaning up the Interacting with ProBuilder section.

https://github.com/Unity-Technologies/com.unity.probuilder/pull/528/files did some of the rest of the cleanup.
This is the rest of the section.

### Links

https://jira.unity3d.com/browse/DOCATT-5807

https://jira.unity3d.com/browse/DOCATT-6407

### Comments to Reviewers

Note https://github.com/Unity-Technologies/com.unity.probuilder/pull/528/files - same area, different work. 

Too many Jira tickets, man.

### CODE REVIEW

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.probuilder?pullRequest=529) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.probuilder?pullRequest=529) before we can accept your contribution.<br/><hr/>**Sue Arkin** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.cds.internal.unity3d.com/check/Unity-Technologies/com.unity.probuilder?pullRequest=529) it.</sub>

File: Documentation~/Editor_Windows.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Editor windows
+
+![Editor windows](images/pb_editors.png)
+
+Editor windows provide [features](tool-panels.md) with extended functionality. For example, the UV Editor window (in the example image above) allows you to perform advanced texture manipulations, including texture mapping, UV unwrapping, and tiling.
```
Comment (JonMUnity) [inline Documentation~/Editor_Windows.md:5]:
```suggestion
Editor windows provide [features](tool-panels.md) with extended functionality. For example, you can use the UV Editor window to perform advanced texture manipulations, including texture mapping, UV unwrapping, and tiling.
```

File: Documentation~/Scene_Information.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Scene Information
+
+![The Scene Information Overlay](images/show-scene-info.png)
```
Comment (JonMUnity) [inline Documentation~/Scene_Information.md:3]:
```suggestion
![The Scene Information overlay](images/show-scene-info.png)
```

File: Documentation~/Scene_Information.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+The **Scene Information** overlay displays information about the selected mesh and selected elements of that mesh.
+
+To display the overly, select the **Show Scene Info** setting in the [Preferences](preferences.md#info_overlay). You [active context](modes.md) must be ProBuilder.
```
Comment (JonMUnity) [inline Documentation~/Scene_Information.md:7]:
```suggestion
To display the Scene Information overlay, select the **Show Scene Info** setting in the [Preferences](preferences.md#info_overlay). Your [active context](modes.md) must be set to ProBuilder to display the Scene Information overlay.
```

File: Documentation~/components.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# ProBuilder mesh components
+
+For a GameObject to be a ProBuilder object, it needs the following components:
+
+* [ProBuilder MeshFilter](ProBuilderMesh.md): All ProBuilder shapes have this components. It displays the **Object Size** of the ProBuilder Mesh in **X**, **Y**, and **Z**. It also allows you to generate lightmap UVs for the Mesh, and customize how ProBuilder generates them. 
+* [PolyShape (script)](polyshape.md): For custom shapes.
+* [Bezier shape (script)](bezier.md) (Experimental)
+* Mesh Renderer and Mesh Collider. These components are standard for 3D objects. For more information, refer to [GameObjects](https://docs.unity3d.com/6000.0/Documentation/Manual/GameObjects.html). 
+
```
Comment (JonMUnity) [inline Documentation~/components.md:9]: "needs" suggests that it *must* have *all* of these. But that is not the case. For example, not every ProBuilder object needs a Bezier shape script. Can we rewrite the introduction to this list?

File: Documentation~/components.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+
+When you first activate a creation tool, Unity adds these components to the new GameObject. They expose specific properties defined in the corresponding scripts which help define the topology. After you create the new Mesh, you can re-activate the tool for the same Mesh and change these properties to modify the Mesh's shape.
```
Comment (JonMUnity) [inline Documentation~/components.md:12]:
```suggestion
When you first activate a creation tool, the Editor adds the appropriate ProBuilder components to the new GameObject. They expose specific properties defined in the corresponding scripts which help define the topology. After you create the new mesh, you can re-activate the tool for the same mesh and change these properties to modify the mesh's shape.
```

File: Documentation~/components.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+> **Note**: When you re-activate one of these tools, you lose any modifications you made to the Mesh through an action or through the [Cut](cut-tool.md) tool. For example, imagine you create a new Poly Shape with five points, and then extrude one of the faces. Next, you decide to remove one of the points, so you enter Poly Shape editing mode again. The extrusion disappears as soon as you re-enter Poly Shape editing mode.
```
Comment (JonMUnity) [inline Documentation~/components.md:14]:
```suggestion
> **Note**: When you re-activate one of these tools, you lose any modifications you made to the mesh through an action or through the [Cut](cut-tool.md) tool. For example, imagine you create a new Poly Shape with five points, and then extrude one of the faces. Next, you decide to remove one of the points, so you enter Poly Shape editing mode again. The extrusion disappears as soon as you re-enter Poly Shape editing mode.
```

File: Documentation~/components.md (hunk 4 above)
Comment (suearkinunity) [reply Documentation~/components.md:9]:
```suggestion
For a GameObject to be a ProBuilder object, it needs the following components:

* [ProBuilder MeshFilter](ProBuilderMesh.md): All ProBuilder shapes have this components. It displays the **Object Size** of the ProBuilder Mesh in **X**, **Y**, and **Z**. It also allows you to generate lightmap UVs for the Mesh, and customize how ProBuilder generates them. 
* [PolyShape (script)](polyshape.md): For custom shapes.
* Mesh Renderer and Mesh Collider. These components are standard for 3D objects. For more information, refer to [GameObjects](https://docs.unity3d.com/6000.0/Documentation/Manual/GameObjects.html). 

```

**Review by JonMUnity — APPROVED** [review]

**Comment by suearkinunity** [comment]
@modrimkus please can you merge this?

**Pull request merged**